CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Change Log
 
+## [1.1.0a1] - 2020-03-27
+
+This release **must** be downloaded via the `get-poetry.py` script and not via the `self update` command.
```

Comment:
I think this should also be documented in the README.md -- I'm afraid many people may not catch it if this information is only in the Changelog.

Reply:
I don't think it's worth it. By the time the `1.1.0` version is out, we will have fixed it in the `self update` command.

And since it's a prerelease version that will not be officially announced on the website, the risk for people to accidentally update to this version is low.